egui_web/src/lib.rs

```diff
@@ -906,11 +952,17 @@ fn install_canvas_events(runner_ref: &AppRunnerRef) -> Result<(), JsValue> {
             let pos = pos_from_touch_event(runner_lock.canvas_id(), &event);
             runner_lock.input.latest_touch_pos = Some(pos);
             runner_lock.input.is_touch = true;
+
+            // TO BE DISCUSSED: todo for all `touch*`-events:
+            // Now that egui knows about Touch events, the backend does not need to simulate
+            // Pointer events, any more.  This simulation could be moved to `egui`.
```

Comment:
I agree - the integrations should either send `Pointer`-events or `Touch`-events, but not both.

Reply:
I agree that this should be done in a separate PR. Should I turn this comment into an issue?

Reply:
Code comment is already removed from PR.